src/mlmc/sampling_pool_pbs.py

```diff
@@ -109,6 +127,7 @@ def pbs_common_setting(self, **kwargs):
                                      '']
 
         self._pbs_header_template.extend(('cd {work_dir}',
+                                          'module load python36-modules-gcc',
```

Comment:
This whole is  Charon specific. We should make it generic and introduce a sort of injection from the PBS configuration.